src/libcore/nonzero.rs

```diff
@@ -62,14 +63,11 @@ impl_zeroable_for_integer_types! {
 /// NULL or 0 that might allow certain optimizations.
 #[lang = "non_zero"]
 #[derive(Copy, Clone, Eq, PartialEq, Ord, PartialOrd, Debug, Hash)]
-pub struct NonZero<T: Zeroable>(T);
+pub struct NonZero<T: Zeroable>(pub(crate) T);
```

Comment:
Is there a reason we need to make this `pub(crate)` instead of using the `new_unchecked` and `get` methods?

Reply:
This is in anticipation for later making this module private and reducing it to its bare minimum (the lang item) https://github.com/SimonSapin/rust/pull/1

Reply:
https://github.com/SimonSapin/rust/blob/private-core-nonzero/src/libcore/nonzero.rs

Reply:
Ah gotcha. I missed that part in the RFC :+1: